src/modules/PublicLab.CustomInsertMaps.js

```diff
@@ -0,0 +1,35 @@
+
+module.exports = function CustomInsert(_module, wysiwyg) {
+  function Syntax(Latitude, Longitude, Layers) {
+    var syn = "[map:content:" + Latitude + ":" + Longitude;
+    if(Layers)
+      syn = syn + ":" + Layers;
+    syn = syn + "]"        
+    return syn;
+  }
+
+  $('.wk-commands').append('<a class="woofmark-command-insert-map btn btn-default" data-toggle="Insert" title="Custom Insert Maps"><i class="fa fa-globe"></i></a>');
+
+  var builder =  '<div class="input-group">';
+  builder += '<input type="number" class="form-control" placeholder="Latitude" id="Latitude" style="min-width: 150px;" required>';
```

Comment:
Hi! I have some similar questions to [this comment](https://github.com/publiclab/PublicLab.Editor/pull/538#pullrequestreview-445890409) about the classnames and ids here. What do you think?